tests/admin/test_views.py

```diff
@@ -0,0 +1,184 @@
+import pytest
+import mock
+import json
+from flask import current_app, redirect, url_for, abort
+from flask_admin import AdminIndexView
+from flask_admin.contrib.fileadmin import FileAdmin as _FileAdmin
+from flask_admin.contrib.pymongo import ModelView as PyMongoModelView
+from flask_simplelogin import is_logged_in
+from quokka.admin.actions import CloneAction, PublishAction
+from quokka.utils.routing import expose
+from quokka.admin.views import RequiresLogin, FileAdmin, IndexView, ModelView
+
+################################
+#pytest - fixtures - setUp();  #
+################################
+rl = RequiresLogin()
+fa = FileAdmin("/home/")
+iv = IndexView() 
+
+
+##################################################
+#pytest - Quokka - quokka/admin/test_views.py  #
+##################################################
+def test_RequiresLogin_class_is_instance_of():
+    assert isinstance(rl, RequiresLogin) == True
+
+def test_FileAdmin_class_instance_of_None_param_base_path():
+    with pytest.raises(TypeError) as err:
+        try:
+            fa = FileAdmin()
+            assert "missing 1 required positional argument:" in str(err.value)
+
+        except OSError as e:
+            if e.errno != errno.EEXIST:
+                raise
+
+        except FileExistsError:
+            raise
+
+        except RuntimeError:
+            raise
+
+        except Exception:
+            raise
+
+def test_FileAdmin_class_instance_of_param_base_path_dont_exists_or_inaccessible():
+    with pytest.raises(OSError) as err:
```

Comment:
Name def function need to be lowercase

Reply:
done => https://github.com/rochacbruno/quokka/pull/667/commits/1a1c4f5d7bb2fd4332bcf39df03497f464b9d98d